cli/azd/pkg/tools/powershell/powershell.go

```diff
@@ -75,5 +91,17 @@ func (bs *powershellScript) Execute(ctx context.Context, path string, options to
 		runArgs = runArgs.WithStdOut(options.StdOut)
 	}
 
-	return bs.commandRunner.Run(ctx, runArgs)
+	result, err := bs.commandRunner.Run(ctx, runArgs)
+	if err != nil {
+		if noPwshError != nil {
+			err = &internal.ErrorWithSuggestion{
+				Err: err,
+				Suggestion: "pwsh (Powershell 7) was not found and powershell (Powershell 5) was automatically used " +
+					"instead. You can try installing pwsh and trying again in case this script is not compatible with " +
+					"Powershell 5.",
```

Comment:
```suggestion
				Suggestion: "pwsh (PowerShell 7) was not found and powershell (PowerShell 5) was automatically used " +
					"instead. You can try installing pwsh and trying again in case this script is not compatible with " +
					"PowerShell 5.",
```